colabs/scikit/wandb_decision_tree.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # Paste your api key here
</pre><p>Just <code>wandb.login()</code> would  suffice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Addressed in [fc10336](https://github.com/wandb/examples/pull/492/commits/fc1033692793f2b7d684cedb5c1f0fc3e549b3c2)